toplevel/native/topmain.ml

```diff
@@ -33,28 +31,13 @@ let expand_position pos len =
     (* New last position *)
     first_nonexpanded_pos := pos + len + 2
 
-
-let prepare ppf =
-  Topcommon.set_paths ();
-  try
-    let res =
-      List.for_all (Topeval.load_file false ppf) (List.rev !preload_objects)
```

Comment:
Self-note: this is bugfix, the native toplevel didn't take in account object files added with OCAMLPARAM.